caluma/caluma_workflow/dynamic_tasks.py

```diff
@@ -0,0 +1,34 @@
+from functools import wraps
+
+name_attr = "_dynamic_task_name"
+
+
+def register_dynamic_task(dynamic_task_name):
+    def wrapper(fn, *arg, **kwarg):
+        setattr(fn, name_attr, dynamic_task_name)
+
+        @wraps(fn)
+        def wrapped(self, *args, **kwargs):
+            return fn(self, *args, **kwargs)
+
+        return wrapped
+
+    return wrapper
+
+
+class BaseDynamicTasks:
```

Comment:
Please add a docstring with an example, as we have it for the other extension base classes.

Reply:
Fair, will do. I copied this from the dynamic groups where it's also missing so it might be clever to add a comment there as well

Reply:
Done

Reply:
I'll add the docstring for the dynamic groups in another PR

Reply:
#1138